sprints/20201218-cross-training/sprint-retrospective.md

```diff
@@ -1,11 +1,49 @@
+## Marco meeting
+
+During our review and retrospective, :mmucci, who has a long-time history and experience in leading Scrums, attended to help us with an opinion from outside the team and provide feedback.
+
+He had several high value suggestions and a somewhat different Sprint model mechanics:
+* In time, it gets better and sprint productivity comes with experience
+* don't have estimates at all as Monte Carlo is used for forecasting
+  - new project comes along, potential MVP
+  - four weeks as normal, people assign tasks and get stuff done
+  - Monte Carlo model is being fed the data and output the speed/velocity
+  - every two weeks we update the forecast (to answer are we on track to finish, ahead/behind?)
+  - people work as normal and that works for them
+  - all the forecasting is done in the background (meet once a week)
+* this model is an efficient method to prevent teams take more work than they can handle
+* forecast helps with setting limits
+* during planning session, Scrum Master sets the MVP e.g. 10 most important bugs, agreed in the team
+* Monte Carlo system can also work with story points
+  - story should not be measured in time. Time itself is the velocity there
+  - possible solution: list of work. Set three buckets: easy, medium, hard. Compare them with each other. It’s not about absolute, but relative. Treat each bucket differently. The easy buckets can have 1-2-3. Randomly pick a bug (medium difficulty and set it to 2). Then compare it against all of the others and challenge. Medium ones are 5 - 8 - 13. Hard ones are 20 and try to break as much as possible. Repeat this every sprint and compare / challenge; it will get better over time.
+
 
 ## What worked
-*
+- Got a lot of stuff done in various different epics, having a good level of cross-training
+- Planning worked well for some of the epics
+- First steps towards tracking interruptduty, though not ideal, a good starting point
+- We actually used a JIRA sprint board and got more familiarized with JIRA mechanics
+- Started having sprint standups in a dedicated channel to save time and knowledge transfer between timezones
+- Started using the story points model; though not fully functional, was a good starting point
 
 ## What didn't
-*
+- Unclear how to split story points into smaller granular tickets < 1 day to account for interrupts or smaller pieces of work
+- Interruptduty was confusing as it wasn't clear what to track and what not
+- Setting up priorities and depicting them in the spring board has not worked properly
+- We need to be more mindful about timezone coverage when sprinting with co-workers
 
 ## Process changes for next sprint
-* {{CHANGE}}
-  * **ACTION**:
+- Have a dedicated channel for daily standup
+- We should allocate good time for planning in order to be successful. We validated that for VPN, we should extend for all epics going forward
+- Set clear number of story points == number of engineering days, rather than adding stuff and trim later.
```

Comment:
I think for sprint 3 we went to the Fibonacci point method to avoid bikeshedding too much about how many story points something should be.

Reply:
Good catch, fixed, thanks!